### PR TITLE
apps/home/measures detail page: adding measures detail page and templ…

### PR DESCRIPTION
**Pull request by khamui**
…ate (draft)

### CODE REVIEW

**Comment by philli-m** [comment]
@khamui just a quick note on structure, I think this should be a separate app, like the gruenbuch app, what do you think? just because it doesn't really belong to home, it will be more like the academy app in liqd site

**Comment by khamui** [comment]
> @khamui just a quick note on structure, I think this should be a separate app, like the gruenbuch app, what do you think? just because it doesn't really belong to home, it will be more like the academy app in liqd site
> 
> NOTE: I added the app here #649 maybe you can review, delete your migrations and rebase on that one? we can also talk about it tomorrow I'll work off mine to do the tiles

@philli-m good point, yes that makes sense :)

File: apps/measures/models.py [hunk 1]
```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
...
+        'responsible_orgs_de_ls',
+    )
+
+    involved_orgs = TranslatedField(
+        'involved_orgs_de',
+        'involved_orgs_en',
+        'involved_orgs_de_ls',
+    )
+
+    date = TranslatedField(
```
Comment (philli-m) [inline apps/measures/models.py:244]: 'date' is a bit odd here, the german translates to maybe more something like 'duration' or something? 
As Max gave them to us in german maybe we should check with him what the form fields should be called in en?

File: apps/measures/models.py [hunk 2]
```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
...
+        'page_title_de_ls',
+    )
+
+    responsible_orgs = TranslatedField(
+        'responsible_orgs_de',
+        'responsible_orgs_en',
+        'responsible_orgs_de_ls',
+    )
+
+    involved_orgs = TranslatedField(
```
Comment (philli-m) [inline apps/measures/models.py:238]: maybe same for below, also as these are seen by editors (who at somepoint may not be max) we can use full length words here

File: apps/measures/templates/apps_measures/measures_detail_page.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    {{ page.contact_name }}<br />
+    {% if page.contact_email %}
+        {% trans 'Email' %}: {{ page.contact_email }}<br />
+    {% endif %}
+    {% if page.contact_phone %}
+        {% trans 'Phone number' %}: {{ page.contact_phone }}<br />
+    {% endif %}
+    {% if page.contact_website %}
+        {% trans 'Website' %}: {{ page.contact_website }}
+    {% endif %}
```
Comment (philli-m) [inline apps/measures/templates/apps_measures/measures_detail_page.html:95]: I would wrap the contact deets in an `<address>` element.
They should also probably be links as makes it more accessible but guess we check with Marcelo first.

File: apps/measures/templates/apps_measures/measures_detail_page.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        <img
+            src="{{ text_image.url }}"
+            alt="{{ text_image.alt }}"
+        />
+        {% if text_image.caption %}
+            <p class="image__caption">{{ text_image.caption }}</p>
+        {% endif %}
+    </div>
+    {{ page.body|richtext }}
+    {% if page.body_participation %}
```
Comment (philli-m) [inline apps/measures/templates/apps_measures/measures_detail_page.html:74]: each of these richtext elements can go into a `<section>` element, I think they have an example of what classes they use in the style guide

File: apps/measures/models.py [hunk 5]
```diff
@@ -10,6 +12,28 @@
...
+    ('4', _('Lichtenberg')),
+    ('5', _('Marzahn-Hellersdorf')),
+    ('6', _('Mitte')),
+    ('7', _('Neukölln')),
+    ('8', _('Pankow')),
+    ('9', _('Reinickendorf')),
+    ('10', _('Spandau')),
+    ('11', _('Steglitz-Zehlendorf')),
+    ('12', _('Tempelhof-Schöneberg')),
+    ('13', _('Treptow-Köpenick')),
```
Comment (philli-m) [inline apps/measures/models.py:34]: just a question, why numbers? is it a database thing?

File: apps/measures/templates/apps_measures/measures_detail_page.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+<div id="layout-grid__area--maincontent">
+    <h1>{{ page.page_title }}</h1>
```
Comment (philli-m) [inline apps/measures/templates/apps_measures/measures_detail_page.html:6]: check styleguide or our code for required classes for headers, think its just 'title' same for all elements on page, img, p included

File: apps/measures/templates/apps_measures/measures_detail_page.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    {% image page.text_image original as text_image %}
+    <div class="meta-information__image">
+        <img
+            src="{{ text_image.url }}"
+            alt="{{ text_image.alt }}"
+        />
+        {% if text_image.caption %}
+            <p class="image__caption">{{ text_image.caption }}</p>
+        {% endif %}
+    </div>
```
Comment (philli-m) [inline apps/measures/templates/apps_measures/measures_detail_page.html:72]: I think this should be in the same section as the description and styleguide has classes for a image in a paragraph

**Review by philli-m — COMMENTED** [review]
Awesome! just a few notes on some wording structure bits as we have to use styleguide as much as possible but otherwise, it's good to go!

File: apps/measures/models.py [hunk 8]
```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
         index.SearchField('page_intro_en', partial_match=True),
         index.SearchField('page_intro_de_ls', partial_match=True),
     ]
+
```
Comment (philli-m) [inline apps/measures/models.py:152]: on line 142 you can add this as a page type

File: apps/measures/models.py [hunk 9]
```diff
@@ -125,3 +149,191 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
...
+    involved_orgs_en = models.CharField(
+        max_length=300, blank=True)
+    involved_orgs_de_ls = models.CharField(
+        max_length=300, blank=True)
+
+    districts = MultiSelectField(
+        blank=True,
+        max_choices=3,
+        choices=BEZIRK_CHOICES
+    )
```
Comment (philli-m) [inline apps/measures/models.py:180]: @khamui sorry just experienced this issue, maybe help text for may choice is good idea?

File: apps/measures/models.py (hunk 5 above)
Comment (khamui) [reply apps/measures/models.py:34]: no, just random. I think from an example.

File: apps/measures/models.py (hunk 9 above)
Comment (khamui) [reply apps/measures/models.py:181]: yes, added. not sure about my wordin, though.

**Review by philli-m — APPROVED** [review]
Super cool!

**Pull request merged**